requirements.txt

```diff
@@ -1,4 +1,5 @@
 gunicorn
-flask-ask>=0.9.8
-kodi-voice>=1.1.1
+git+git://github.com/johnwheeler/flask-ask@master#egg=flask-ask
+#kodi-voice>=1.1.1
+git+git://github.com/succo69/kodi-voice@master#egg=kodi-voice
```

Comment:
Is there something wrong with the original kodi-voice and flask-ask packages?

Reply:
well, yes... flask-ask 0.9.8 doesn't have the fix to be installed with pip > 9.0.3 (because it calls pip.req) and the same is with 'plain' kodi-voice, which contains 
`from pip.req import parse_requirements`
(which seems to be unused within setup.py)
without these fixes the source doesn't pass travis check, i used my fork of kodi-voice and the git master for flask-ask because the fix isn't in the current (public) version

Reply:
If you can submit a PR to flask-ask to fix it. I can approve the PR.

Reply:
But flask-ask already contains that fix in master, or am I pointing to the wrong flask-ask repo?

Reply:
Oh, I see what you mean. Yeah, I can merge PR's on that repo, but I can't update the package on PyPI. Good call.

Reply:
btw, where did flask-ask 0.9.8 come from? it seems to be behind PyPI version (johnwheeler shows 0.9.7 in setup.py)
i guess we need to point to the github and not to the PyPI versions to avoid errors during the installation